Packages/Env/Sources/Env/KeychainHelper.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+public enum KeychainHelper {
```

Comment:
We already gave KeychainSwift in the app. I don't think you need this. You can look at how AppAccount is done and maybe use the same library and pattern?